how-it-works/2_chainkeytech/04_chain_key_tokens.subpage.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Chain-key Tokens
+abstract:
+shareImage: /img/how-it-works/direct-integration-with-bitcoin.600.jpg
+slug: chain-key-tokens
+---
+
+# Chain-Key Tokens
+
+One important way how different blockchains can cooperate is by bringing tokens from one blockchain to another one, e.g., bringing Bitcoin to the Ethereum chain. In traditional blockchain architectures, this is mostly accomplished through wrapping. More decentralized (and secure) alternatives to wrapping exist and can replace it: Meet *chain-key cryptography* and *chain-key tokens* as explained on this page!
```

Comment:
I think it sounds better when removing "as explained on this page".
Isn't it clear that these concepts are discussed here?

Reply:
OK